src/providers/VoiceFocusProvider/index.tsx

```diff
@@ -21,6 +22,7 @@ interface Props {
    * For more info, you can go to https://aws.github.io/amazon-chime-sdk-js/interfaces/voicefocusdeviceoptions.html
    */
   options?: VoiceFocusDeviceOptions;
+  createMeetingResponse?: any;
```

Comment:
Could we avoid `any`?

Reply:
We have a type for this, and we should use it.

Reply:
The type is removed. Same for JS SDK. https://code.amazon.com/packages/ChimeNoiseSuppressionWeb/commits/15e3f7ad06598e5beb9b98c06aae046647a9c2eb#
https://github.com/aws/amazon-chime-sdk-js/blob/147e7bb8f0b38e13410a50956c921d54cd872f44/src/voicefocus/VoiceFocusDeviceTransformer.ts#L138

Reply:
lets not post the code.amazon.com links - those are internal.